### PR TITLE
Update contributing docs and remove lint from makefile

### PR DESCRIPTION
**Pull request by FabioRosado**
This PR adds a bit more information on how to develop locally, it mentions activating the conda env so pre-commit and all the rest of the dependencies are installed and also how to install pre-commit (are you happy with this or would you rather run it manually?)

I've also removed the mention that you needed to rebase all the commits on your branch into one (it seems we are okay with multiple commits since we are squash merging? Happy to revert the change though :smile: )

closes: #770

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,10 +78,87 @@ If you have questions about the project, using PyScript, or anything else, pleas
...
+```
+    git push origin main
+```
+
+* Activate the conda environment (this environment will contain all the necessary dependencies)
+
+```
+    conda activate pyscriptjs/env/
+```
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; **NOTE**: We are assuming you are in the root folder. If you are in the pyscriptjs you can run `conda activate env/` instead.
```
Comment (marimeireles) [inline CONTRIBUTING.md:103]: Is that how it goes if you follow the `make setup` instructions?
It creates an  environment named `env`?

**Review by marimeireles — COMMENTED** [review]
Cool! I love it! :D
Thank you so much for getting this @FabioRosado.
Soo... if the answer to my question is a yes, than this PR is approved :)
If not, I think we should make it generic like `conda install <your env name>`.
Cheers!

File: CONTRIBUTING.md (hunk 1 above)
Comment (FabioRosado) [reply CONTRIBUTING.md:103]: Yeah, I think its because the `environment.yml` file doesn't contain `name: pyscript`? 🤔

**Review by marimeireles — APPROVED** [review]

**Comment by marimeireles** [comment]
Thank you so much @FabioRosado, it already helped at least one person, surely it will help many others ✨

**Pull request merged**